config/database.yml

```
@@ -55,7 +55,7 @@ development:
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
-  url: <%= Settings.database_url %>
+  url: <%= Settings.test_database_url || Settings.database_url %>
```

Now that you've defined `Settings.test_database_url` in our settings, should this just be
```suggestion
  url: <%= Settings.test_database_url %>
```
?